BitwardenShared/Core/Vault/Helpers/VaultListSectionsBuilder.swift

```diff
@@ -298,5 +309,6 @@ struct VaultListPreparedData {
     var foldersCount: [Uuid: Int] = [:]
     var groupItems: [VaultListItem] = []
     var noFolderItems: [VaultListItem] = []
+    var restrictedOrganizationIds: [String] = []
```

Comment:
⛏️ I think in this case I'd add a comment to indicate that this has to do with the `.restrictItemTypes` policy